[{"message":"Cycle found in tasks (dependency graph): task-with-result -> task-with-resources-union -> task-with-result","rule":"no-pipeline-task-cycle","level":"error","path":"./regression-tests/general/task-dependency-cycles-combined.yaml","loc":{"range":[355,371,372],"startLine":17,"startColumn":13,"endLine":17,"endColumn":29}},{"message":"Cycle found in tasks (dependency graph): task-with-result -> task-with-resources-union -> task-with-result","rule":"no-pipeline-task-cycle","level":"error","path":"./regression-tests/general/task-dependency-cycles-combined.yaml","loc":{"range":[125,150,151],"startLine":8,"startColumn":13,"endLine":8,"endColumn":38}},{"message":"Pipeline 'combined-task-with-union-edge' references task 'my-task-with-resources' but the referenced task cannot be found. To fix this, include all the task definitions to the lint task for this pipeline.","rule":"no-missing-resource","level":"error","path":"./regression-tests/general/task-dependency-cycles-combined.yaml","loc":{"range":[180,202,203],"startLine":10,"startColumn":15,"endLine":10,"endColumn":37}},{"message":"Pipeline 'combined-task-with-union-edge' references task 'my-task-with-results' but the referenced task cannot be found. To fix this, include all the task definitions to the lint task for this pipeline.","rule":"no-missing-resource","level":"error","path":"./regression-tests/general/task-dependency-cycles-combined.yaml","loc":{"range":[401,421,422],"startLine":19,"startColumn":15,"endLine":19,"endColumn":35}}]